src/Presentation/Console/Command/ShowWorkflowCommand.php

```diff
@@ -42,9 +42,9 @@ class ShowWorkflowCommand extends Command
     protected ProjectWorkflow $projectWorkflow;
 
     /**
-     * @var \SprykerSdk\Sdk\Infrastructure\Service\CliValueReceiver
+     * @var \SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver\CliReceiver
      */
-    protected CliValueReceiver $cliValueReceiver;
+    protected CliReceiver $cliValueReceiver;
```

Comment:
It must be depended on the abstraction. Please change it to the `ReceiverInterface` interface.